### PR TITLE
Cardinal directions

### PR DESCRIPTION
**Pull request by tzufgoogle**
Added cardinal direction from start point to end point.

E.g., "Starting at Adda Coffee & Tea House walk North-East past Allegheny Sandwich Shop and your goal is Badamo's Pizza, near El Burro. If you reached Rasta House, you have gone too far."

- Build, test and run working.

### CODE REVIEW

File: cabby/geo/item.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ class RVSPath:
   main_pivot: Dict = attr.ib()
   near_pivot: Dict = attr.ib()
   beyond_pivot: Dict = attr.ib()
+  cardinal_direction: Text = attr.ib()
```
Comment (jasonbaldridge) [inline cabby/geo/item.py:40]: Need to add this to the class documentation. Please include some indication of what possible values are -- e.g. is it just north, south, east, and west, or does it include south-west, etc?

**Review by jasonbaldridge — CHANGES_REQUESTED** [review]
I think we should move this into cabby/rvs and also separate the instruction code out, e.g. by providing a method describe_rvs_path() in cabby/rvs/speak.py. As it is now, we are entangling data representation with instruction generation, which means we'll be maintaining separate instruction generation code and lose opportunities for better abstractions and testing.

Please also update the PR description to give examples of the instructions supported by the change to give intuitive context for the reviewer. (Already known in this case, but will be good practice for future PR's!)

File: cabby/geo/item.py (hunk 1 above)
Comment (tzufgoogle) [reply cabby/geo/item.py:42]: Added cardinal description and possible values.

Comment (tzufgoogle) [reply cabby/geo/item.py:42]: Added cardinal description and possible values.

**Review by tzufgoogle — COMMENTED** [review]
Moved the item to rvs and added cardinal description and possible values.

File: cabby/geo/map_processing/map_structure.py [hunk 2]
```diff
@@ -98,6 +99,8 @@ def get_poi(self) -> Tuple[GeoSeries, GeoSeries]:
     osm_poi_streets = osm_poi_named_entities[osm_highway.notnull()]
 
     # Get centroid for POI.
+    osm_poi_no_streets = osm_poi_no_streets.set_crs(epsg=OSM_CRS, 
+    allow_override=True)
```
Comment (jasonbaldridge) [inline cabby/geo/map_processing/map_structure.py:103]: The tabbing seems weird here... can you check and fix?

**Review by jasonbaldridge — APPROVED** [review]
LGTM -- just need to check one weird tabbing issue I noted.

**Pull request merged**